modules/videopress/class.jetpack-videopress.php

```diff
@@ -109,6 +110,21 @@ public function is_connection_owner( $user_id = false ) {
 		return $user_token && is_object( $user_token ) && isset( $user_token->external_user_id ) && $user_id === $user_token->external_user_id;
 	}
 
+	public function add_media_new_notice()
+    {
+	    global $pagenow;
+
+	    if ( $pagenow != 'media-new.php' ) {
+	        return;
+        }
+
+        ?>
+        <div id="message" class="notice notice-warning">
+            <p></p><?php echo sprintf( __( 'If you would like to upload a video to VideoPress, please use the <a href="%s">Media Library</a> instead.', 'jetpack' ), '/wp-admin/upload.php' ); ?></p>
```

Comment:
Also, rather than talk about "VideoPress", let's keep this generic and make the message something like:

"You can upload video via your [Media Library], or when creating a [new post]."